src/main/resources/messages_en_US.properties

```diff
@@ -945,6 +945,7 @@ watermark.selectText.6=heightSpacer (Space between each watermark vertically):
 watermark.selectText.7=Opacity (0% - 100%):
 watermark.selectText.8=Watermark Type:
 watermark.selectText.9=Watermark Image:
+watermark.selectText.10=Convert PDF to PDF-Image
```

Comment:
Default Lang file is UK not US can you please add there as well!

Reply:
Sorry, I didn't know we use proper English here ;)
Let me fix that